docs/reference/typescript-sdk/transfers.mdx

```diff
@@ -0,0 +1,81 @@
+# Transfers
+
+## Fee Estimation
+
+When sending a cross-chain transfer with Hyperlane GMP, two fees are required:
+
+1. **Local Fee** – gas to submit the transaction on the origin chain
+2. **Interchain Fee** – gas to deliver and execute the message on the destination chain
+
+Both are paid upfront to the origin chain.
+
+This fee structure ensures that:
+
+1. The user has enough funds to pay for the local transaction
+2. The cross-chain message will have enough gas to execute on the destination chain and pay for the message delivery
+3. The total cost of the transfer is transparent to the user before execution
+
+#### Fee Token Considerations
+
+When sending cross-chain transfers, the fee tokens vary by chain type:
+
+1. **EVM Chains**
+
+   - Uses native token (e.g., ETH) for both local and interchain fees
+
+2. **Sealevel (Solana) Chains**
+
+   - Uses native SOL for transaction fees
+
+3. **Cosmos Chains**
+
+   - Supports custom fee tokens
+   - Fees are specified using the `denom` parameter
+
+### Estimating Fees
+
+To estimate both fees, use `estimateTransferRemoteFees`:
+
+```ts
+const { interchainQuote, localQuote } =
+  await warpCore.estimateTransferRemoteFees({
+    originToken,
+    destination,
+    sender,
+  });
+```
+
+- `interchainQuote`: Fee to route the message to the destination chain (via IGP)
+- `localQuote`: Gas cost to send the tx on the origin chain
```

Comment:
```suggestion
- `originQuote`: Gas cost to send the tx on the origin chain
```

Reply:
since ts sdk called localquote leaving this as is: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/1482467faf063b67d2dcbaba31d08ed06f8bf785/typescript/sdk/src/warp/WarpCore.ts#L411